addon/components/docs-viewer/x-main/component.js

```diff
@@ -3,7 +3,7 @@ import Component from '@ember/component';
 import layout from './template';
 import { computed } from '@ember/object';
 import appFiles from 'ember-cli-addon-docs/app-files';
-import { dasherize } from '@ember/string';
+import addonFiles from 'ember-cli-addon-docs/addon-files';
```

Comment:
Needed to add the logic to also expose the `addon` tree contents, otherwise you had to really guess a lot about what the current URL on Github would be. At least now we have a reference of the actual files to try and match against.